tests/tck/features/go/GoYieldVertexEdge.feature

```diff
@@ -54,13 +54,6 @@ Feature: Go Yield Vertex And Edge Sentence
       | ("LaMarcus Aldridge" :player{age: 33, name: "LaMarcus Aldridge"}) | "teammate" |
       | ("Manu Ginobili" :player{age: 41, name: "Manu Ginobili"})         | "teammate" |
       | ("Tony Parker" :player{age: 36, name: "Tony Parker"})             | "teammate" |
-      | ("Manu Ginobili" :player{age: 41, name: "Manu Ginobili"})         | "like"     |
```

Comment:
After discussing with @MuYiYong  offline, the incompatibility of this situation is acceptable